orchestra/migrations/0074_auto_20180112_2047.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
If easy, can you provide screenshots of the TODO list?

Reply:
Done. :D